scripts/tests/test_validate_docstrings.py

```diff
@@ -706,6 +735,11 @@ def test_bad_generic_functions(self, func):
         ('BadGenericDocStrings', 'private_classes',
          ("Private classes (NDFrame) should not be mentioned in public "
           'docstrings',)),
+        ('BadGenericDocStrings', 'unknown_section',
+         ('Found unknown section "Unknown Section".',)),
+        ('BadGenericDocStrings', 'sections_in_wrong_order',
+         ('Wrong order of sections. "See Also" should be located before '
+          '"Notes"',)),
```

Comment:
To make sure I understand the error message, we say "See Also should be located before Notes" because it's the section immediately preceding notes. But the "users" docstring in this case doesn't have a "Notes" section, so that's slightly confusing.

I was hoping we could instead say "See Also should be located before Examples", but that's not quite right either, because if you have three out of order sections, you would want to skip back two sections, if that makes sense...

So I guess my only comment is, "this error message is potentially confusing. But I don't know how to improve it". If you see an easy way to improve it, push it here. Otherwise +1 from me.

Reply:
> But I think it should be all right, as in the error message I show the order of all sections (is not shown in the test, but will be displayed to the users).

Ah great. That sounds fine.